docs/tutorial.rst

```diff
@@ -0,0 +1,288 @@
+    >>> s  # doctest: +REMOTE_DATA
+                                     Dopplergram
+    0  /SUM58/D803708321/S00008/Dopplergram.fits
+    1  /SUM41/D803708361/S00008/Dopplergram.fits
+    2  /SUM71/D803720859/S00008/Dopplergram.fits
+    3  /SUM70/D803730119/S00008/Dopplergram.fits
+
+Note that the ``key`` and ``seg`` parameters can also be used together in one :func:`Client.query` call, i.e.::
+
+    >>> k, s = c.query('hmi.v_45s[2016.04.01_TAI/1d@6h]',
+    ...                key='T_REC, DATAMEAN, OBS_VR', seg='Dopplergram')  # doctest: +REMOTE_DATA
+
+The file paths listed above are the storage location on the JSOC server.
+You can access these files, even if you do not have direct NFS access to the filesystem, by prepending the JSOC URL to segment file path::
+
+    >>> url = 'http://jsoc.stanford.edu' + s.Dopplergram[0]  # doctest: +REMOTE_DATA
+    >>> url  # doctest: +REMOTE_DATA
+    'http://jsoc.stanford.edu/SUM58/D803708321/S00008/Dopplergram.fits'
+
+    >>> from astropy.io import fits
+    >>> a = fits.getdata(url)  # doctest: +REMOTE_DATA
+    >>> print(a.shape, a.dtype)  # doctest: +REMOTE_DATA
+    (4096, 4096) float32
+
+Note that FITS files which are accessed in this way, do not contain anykeyword data in their headers.
+This is perfectly fine in many cases, because you can just use :func:`Client.query` to obtain the data of all required keywords.
+If you need FITS files with headers that contain all the keyword data, you need to submit an export request to JSOC, which is described in the next section.
+
+Export requests can also be useful, if you want to download more than only one or two files (even without keyword headers), because you can then use the :func:`ExportRequest.download` method, which takes care of creating URLs, downloading the data and (if necessary) generating suitable local filenames.
+
+Data export requests
+--------------------
+
+Data export requests can be interactively built and submitted on the `JSOC Export Data <http://jsoc.stanford.edu/ajax/exportdata.html>`__ webpage, where you can also find more information about the different export options that are available.
+Note that a registered email address is required to for submitting export requests. You can register your email address on the `JSOC email registration <http://jsoc.stanford.edu/ajax/register_email.html>`__ webpage.
+
+It is advisable to have a closer look at the export webpage before submitting export requests using the ``drms`` library.
+It is also possible to submit an export request on the webpage and then use the Python routines to query the request status and download files.
+
+First, we start again with importing the ``drms`` library and creating a `drms.Client` instance::
+
+.. warning::
+    Please replace the email below with your own registered email.
+
+    >>> import drms
+    >>> c = drms.Client(email='nabil.freij@gmail.com', verbose=True)  # doctest: +REMOTE_DATA
```

Comment:
I am not sure what I should replace this with.

Reply:
`sunpy@sunpy.org`? As in here: https://docs.sunpy.org/en/stable/api/sunpy.net.jsoc.JSOCClient.html#jsocclient

Reply:
Done.